modules/imgproc/src/hough.cpp

```diff
@@ -913,6 +913,17 @@ void HoughLines( InputArray _image, OutputArray lines,
         HoughLinesSDiv(_image, lines, type, (float)rho, (float)theta, threshold, cvRound(srn), cvRound(stn), INT_MAX, min_theta, max_theta);
 }
 
+void HoughLinesAcc(InputArray _image, OutputArray lines,
```

Comment:
I believe we should not expose this through exported C++ API.

This should work: "static inline" function with `CV_WRAP_AS(HoughLinesWithAccumulator)`

(without `CV_INSTRUMENT_REGION();`)